docs/src/whatsnew/latest.rst

```diff
@@ -65,7 +71,10 @@ This document explains the changes made to Iris for this release
 🔥 Deprecations
 ===============
 
-#. N/A
+#. `@hsteptoe`_ and `@trexfeathers`_ (reviewer) deprecated
```

Comment:
@trexfeathers  Just checking... does this cover your comment [above](https://github.com/SciTools/iris/pull/4669/files/bff3e3afaede61806cb99f65359c4cf4e49ff90c#r1003335525)

Reply:
All good 👍